guides/inference-scheduling/gaie-inference-scheduling/values.yaml

```diff
@@ -5,24 +5,24 @@ inferenceExtension:
     ###################
     name: llm-d-inference-scheduler
     hub: ghcr.io/llm-d
-    tag: v0.2.1
+    tag: v0.3.0-rc.2
     ###################
     # name: epp
     # hub: registry.k8s.io/gateway-api-inference-extension
-    # tag: v0.5.1
+    # tag: v1.0.0-rc.4
     ###################
     pullPolicy: Always
   extProcPort: 9002
-  # The plugins-v2.yaml contains the approximate prefix aware plugin configuration.
-  # https://github.com/kubernetes-sigs/gateway-api-inference-extension/blob/978c23bb96cce25f11cea2421523cb479026e2e9/config/charts/inferencepool/templates/epp-config.yaml#L57
-  # This will become the default in the next gaie release.
-  pluginsConfigFile: "plugins-v2.yaml"
+  pluginsConfigFile: "default-plugins.yaml"
+  flags:
+    - name: "pool-group"
+      value: "inference.networking.x-k8s.io"
 inferencePool:
-  targetPortNumber: 8000
+  targetPortNumber: 8000 # THIS IS GKE ONLY
+  targetPorts:
```

Comment:
I don’t think this is needed when using the previous crd

Reply:
I thought we need it for the `inferencepool` here: https://github.com/kubernetes-sigs/gateway-api-inference-extension/blob/v1.0.0/config/charts/inferencepool/templates/inferencepool.yaml#L31-L34 _specifically_ when using the old API version

Reply:
the first line in this file is `if` check. 
we check if the apiVersion is v1alpha2 here:
https://github.com/kubernetes-sigs/gateway-api-inference-extension/blob/ffea26329a16922bb3b2cf37a383417c511227d5/config/charts/inferencepool/templates/inferencepool.yaml#L1

if yes, we use the v1alpha2 format.

in line 21 we have `else` (only if apiVersion is different than v1alpha2, which basically means it's v1):
https://github.com/kubernetes-sigs/gateway-api-inference-extension/blob/v1.0.0/config/charts/inferencepool/templates/inferencepool.yaml#L21

That basically means that we don't use the `targetPorts` when using `v1alpha2` apiVersion.

Reply:
Sounds great I must have missread that in the templates. Will remove this targetPorts then